sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClientTest.java

```diff
@@ -282,6 +285,65 @@ void sendMultipleMessages() {
         messagesSent.forEach(message -> Assertions.assertEquals(Section.SectionType.Data, message.getBody().getType()));
     }
 
+    /**
+     * Verifies that sending multiple message will result in calling sender.send(Message...).
+     */
+    @Test
+    void sendMessagesList() {
+        // Arrange
+        final int count = 4;
+        final byte[] contents = TEST_CONTENTS.getBytes(UTF_8);
+        final ServiceBusMessage[] messages = new ServiceBusMessage[count];
+
+        IntStream.range(0, count).forEach(index -> {
```

Comment:
Why not `for (int i = 0; i < messages.length; i++)`? I don't see any added benefit creating this IntStream.